constants.py

```diff
@@ -1,6 +1,12 @@
+# useful constants during analysis
 
 FEATURES = [
     'CIRCLE', 'SQUARE', 'STAR', 'TRIANGLE', 
     'CYAN', 'GREEN', 'MAGENTA', 'YELLOW', 
     'ESCHER', 'POLKADOT', 'RIPPLE', 'SWIRL'
-]
\ No newline at end of file
+]
+
+NUM_UNITS = 59
```

Comment:
Shouldn't this something you can infer from the dataset size?

Reply:
ah yea, just got lazy here. Fixed.